include/oneapi/dpl/pstl/utils.h

```diff
@@ -717,6 +717,41 @@ struct __spirv_target_conditional :
 // to determine SPIR-V targets.
 inline constexpr bool __is_spirv_target_v = __spirv_target_conditional<::std::true_type, ::std::false_type>::value;
 
+// Lower bound implementation based on Shar's algorithm for binary search.
+template <typename _Acc, typename _Size1, typename _Value, typename _Compare>
+_Size1
```

Comment:
Is there a reason to use the name `_Size1` instead of `_Size`? I don't see a `_Size2`.

Reply:
No, I just followed what was being done in `__pstl_lower_bound`. It is confusing, so I have changed it to `_Size`.